tests/unit/python/fledge/common/test_configuration_manager.py

```diff
@@ -2842,3 +2828,30 @@ def test_good__validate_value_per_optional_attribute(self, new_value_entry, stor
         except Exception:
             raised = True
         assert raised is False
+
+
+    async def test__ignore_not_supported_key_in_config_items(self):
+
+        async def mock(return_value):
+            return return_value
+
+        storage_client_mock = MagicMock(spec=StorageClientAsync)
+        c_mgr = ConfigurationManager(storage_client_mock)
+        CAT_NAME = 'test'
+        entry_name = "test_12"
+        item_name = "ignore_entry_name"
+        test_config = {
+            item_name: {    
+                "description": "Test with entry_name",
+                "type": "string",
+                "default": "test_default_value",
+                entry_name: "test_with_invalid_value"
+            }
+        }
+        with patch.object(_logger, 'warning') as wlog:
+            # with patch.object(ConfigurationManager, '_validate_category_val', return_value=mock(item_name)) as val_patch:
+            await c_mgr._validate_category_val(CAT_NAME, test_config)
+            assert 1 == wlog.call_count
```

Comment:
Shift left of this assertion statement. It should be in same level of context patch